scaleway/data_source_account_ssh_key_test.go

```diff
@@ -1,48 +1,54 @@
 package scaleway
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
+const dataSourceAccountSSHKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAILHy/M5FVm5ydLGcal3e5LNcfTalbeN7QL/ZGCvDEdqJ foobar@example.com"
```

Comment:
can you scope this variable in the test so we are sure nobody can use the same SSH key in two tests and avoid concurency